strings/remove_duplicate.py

```diff
@@ -1,14 +1,15 @@
-""" Created by sarathkaul on 14/11/19 """
-
-
 def remove_duplicates(sentence: str) -> str:
     """
-    Reomove duplicates from sentence
+    Remove duplicates from sentence
     >>> remove_duplicates("Python is great and Java is also great")
     'Java Python also and great is'
+    >>> remove_duplicates("Python   is      great and Java is also great")
+    'Java Python also and great is'
     """
-    return " ".join(sorted(set(sentence.split(" "))))
+    return " ".join(sorted(set(sentence.split())))
 
 
 if __name__ == "__main__":
-    print(remove_duplicates("INPUT_SENTENCE"))
```

Comment:
Why remove this line?  Without it, the user runs the program and nothing happens.

Reply:
I think algorithm should not print anything. Because it have tested it using doctest.

Reply:
The algorithmic function does not print() in alignment with CONTRIBUTING.md.  The algorithmic function instead returns a string.  However, it is more instructive (and intuitive) for the reader if main() does print the result of the algorithm.  We are writing for humans to learn, not fmerely or computers to test.